src/Control/Eff/QuickStart.hs

```diff
@@ -21,6 +22,14 @@
 -- import Control.Eff.Exception
 -- @
 --
+-- also, some language extensions are required
+--
+-- @
+-- {-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
@power-fungus same observation here. in the generated docs i see
```
also, some language extensions are required

{--}
{--}
{--}
```

Reply:
This is fixable by `\`-escaping the `#` symbols